crypto/cms/cms_ess.c

```diff
@@ -335,3 +338,57 @@ ASN1_OCTET_STRING *cms_encode_Receipt(CMS_SignerInfo *si)
     CMS_ReceiptRequest_free(rr);
     return os;
 }
+
+/* Add a signing certificate's digest to a SignerInfo */
+
```

Comment:
Remove this blank line.  Make the comment be a multi-line comment (not required, but useful to help mark off the start of a function).  Replace "a SignerInfo" with the variable names, using pipe symbols to mark the name like ```|signer|```.